addons/binding/org.openhab.binding.mysensors/src/main/test/org/openhab/binding/mysensors/test/DiscoveryServiceTest.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mysensors.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.openhab.binding.mysensors.MySensorsBindingConstants.THING_UID_MAP;
+
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.junit.Test;
+
+public class DiscoveryServiceTest {
+
+    @Test
+    public void testThingUidMap() {
+        Set<Integer> keySet = THING_UID_MAP.keySet();
+
+        for (Integer key : keySet) {
```

Comment:
You should walk straight over entrySet (instead of over the keySet and using get on each key) but for test the performance difference is not that relevant.